soundconverter/gstreamer/discoverer.py

```diff
@@ -124,6 +124,15 @@ def _analyse_file(self, sound_file):
             sound_file.info = info
 
             taglist = info.get_tags()
+
+            if not taglist:
+                # fallback
+                # https://bugs.launchpad.net/soundconverter/+bug/1945838
+                taglist = info.get_audio_streams()[0].get_tags()
+
+            if not taglist:
+                return
+
             taglist.foreach(lambda *args: self._add_tag(*args, sound_file))
```

Comment:
@kassoulet what exactly is the bug that was fixed with https://github.com/kassoulet/soundconverter/commit/8bf0b18a9dd69a52b6a212b3b1d9c713defab09c?

if the bug is just because of get_tags, wouldn't it also work like this?

```py
            taglist = info.get_tags()

            if not taglist:
                # fallback
                # https://bugs.launchpad.net/soundconverter/+bug/1945838
                taglist = info.get_audio_streams()[0].get_tags()

            if taglist:
              taglist.foreach(lambda *args: self._add_tag(*args, sound_file))
```